definitions/aiops-workflow/definition.yml

```diff
@@ -0,0 +1,5 @@
+domain: AIOPS
+type: WORKFLOW
+configuration:
+  entityExpirationTime: MANUAL
+  alertable: true
```

Comment:
This entity is not actually alertable. But if I understood the advice from Entity team correctly, this property should be set to true for the entity to be visible in the entity finder.